includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -1018,4 +1048,31 @@ public function get_content( int $post_id = 0, int $return_length = 0, bool $use
 		return apply_filters( 'classifai_chatgpt_content', $content, $post_id );
 	}
 
+	/**
+	 * This function should return language for prompt.
+	 *
+	 * @since x.x.x
+	 * @param string $language_code Language code.
+	 */
+	private function get_language_for_prompt( string $language_code ): string {
```

Comment:
I almost think most of this should be moved into a shared utility function (so outside of ChatGPT and OpenAI). I say this because I can imagine other providers/features wanting to be able to get the full language name from a language code.

I would move that portion of this code into a shared helper function and than this method can then just call that utility function and return the `" in $language_english_name language"` part, if that makes sense